src/test/java/tests/SignInValidationTest.java

```diff
@@ -4,24 +4,25 @@
 import org.testng.Assert;
 import org.testng.annotations.Test;
 import fragments.SignInFragment;
+import org.testng.asserts.SoftAssert;
 
 public class SignInValidationTest extends BaseTest {
 
     private static final String INVALID_EMAIL = "fgjhj";
     private static final String INVALID_PASSWORD = "46trdgdfg";
     private static final String EXPECTED_ERROR_MESSAGE = "Please enter a valid email address.";
+    SoftAssert softAssert = new SoftAssert();
```

Comment:
There was only 1 check here, so you could have left hard assertion, but since you decided to use soft assertion everywhere, maybe you should move its instance to BaseTest?